common/validation/SignUpValidation.js

```diff
@@ -26,6 +26,7 @@ module.exports = [
     .not()
     .isEmpty()
     .withMessage(validationMessage.required)
-    .isLength({ min: 5 })
-    .withMessage(validationMessage.min(5))
+    .matches(/^(?=(.*?[A-Z]){3,})(?=(.*[a-z]){3,})(?=(.*[\d]){3,})(?=(.*[\W]){3,})(?!.*\s).{8,}$/)
```

Comment:
I want like this.

![IMG_20220319_170714.jpg](https://user-images.githubusercontent.com/46679050/159122285-efe5c5da-89e6-436e-9595-fd2c41563fa2.jpg)